src/meltano/core/tracking/tracker.py

```diff
@@ -44,6 +51,17 @@ def check_url(url: str) -> bool:
     return bool(re.match(URL_REGEX, url))
 
 
+class AnalyticsSettings(NamedTuple):
```

Comment:
Wdyt about calling this class `Telemetry settings`?

Reply:
Sure! Initially it was named after the analytics.json file, but TelemetrySettings seems more appropriate, I think. 👍